### DISCUSSION TITLE
Bing CAPTCHA

### DISCUSSION THREAD

**Discussion by andresayac** [Richard-Weiss/node-chatgpt-api#18] [General]
How are you doing to solve the CAPTCHA? I would like to know.

**Comment by Richard-Weiss** [comment]
@andresayac  It should be sufficient to send a message in the Bing or Copilot app or website.
If that doesn't work, you can check this issue:
https://github.com/waylaidwanderer/node-chatgpt-api/issues/439

**Reply by andresayac** [reply]
If I'm doing it manually that way, I wanted to see if they had something automatic for it. I've been thinking about using puppeter or something similar to try to solve it when I have that problem.

thanks @Richard-Weiss

**Reply by Richard-Weiss** [reply]
@andresayac
I'm not sure if that would work.
It's not like old CAPTCHA methods, as it tracks different metrics when clicking the box.
It happens quite rarely, like every two weeks or so, so I've never looked into that.

**Reply by andresayac** [reply]
> @andresayac No estoy seguro de si eso funcionaría. No es como los antiguos métodos CAPTCHA, ya que rastrea diferentes métricas al hacer clic en el cuadro. Ocurre muy raramente, cada dos semanas aproximadamente, así que nunca lo he investigado.

**Reply by PeronGH** [reply]
Actually SydneyQt has already implemented auto captcha solving using https://github.com/Harry-zklcdc/go-bingai-pass

**Reply by andresayac** [reply]
I understand that when mounting it, it leaves a port up and the other script consults it to solve the captcha, for now I mount the api I will study the parameters that I should send, thank you very much @PeronGH
